src/components/JobWizard/JobWizardSteps/JobWizardHyperparameterStrategy/JobWizardHyperparameterStrategy.js

```diff
@@ -66,24 +68,24 @@ const JobWizardHyperparameterStrategy = ({ formState }) => {
         </div>
         <div className="form-table-title ranking-title-grid-item">Ranking</div>
         <div className="result-grid-item">
-          <FormInput label="Result" name={resultPath} />
+          <FormInput id="hyperparams-ranking-result" label="Result" name={resultPath} />
         </div>
         <div className="criteria-grid-item">
-          <FormSelect label="Criteria" name={criteriaPath} options={selectOptions.criteria} />
+          <FormSelect id="hyperparams-ranking-criteria" label="Criteria" name={criteriaPath} options={selectOptions.criteria} />
         </div>
         <div className="form-table-title stop-condition-title-grid-item">Stop condition</div>
         <div className="stop-condition-grid-item">
-          <FormInput label="Stop condition" name={stopConditionPath} />
+          <FormInput id="hyperparams-stop-condition" label="Stop condition" name={stopConditionPath} />
         </div>
         <div className="form-table-title parallelism-title-grid-item">Parallelism</div>
         <div className="parallel-runs-grid-item">
-          <FormInput label="Parallel runs" name={parallelRuns} type="number" min="0" />
+          <FormInput id="hyperparams-parallel-runs" label="Parallel runs" name={parallelRuns} type="number" min="0" />
         </div>
         <div className="dask-cluster-uri-grid-item">
-          <FormInput label="Dask clutter URL" name={dashClusterUriPath} />
+          <FormInput id="hyperparams-dask-url" label="Dask clutter URL" name={dashClusterUriPath} />
         </div>
         <div className="teardown-dask-grid-item">
-          <FormCheckBox label="Teardown" name={teardownDask} />
+          <FormCheckBox id="hyperparams-teardown" label="Teardown" name={teardownDask} />
```

Comment:
Can be applied the same logic as in the FormInput (using name prop inside the component as an id)

Reply:
It is the same logic. I don't use the "name" prop for the ID since I don't want the ID with "." char.